zulipterminal/api_types.py

```diff
@@ -14,6 +14,7 @@ class Message(TypedDict, total=False):
     timestamp: int
     client: str
     subject: str  # Only for stream msgs.
+    subject_links: List[str]  # To support Zulip 2.1+
```

Comment:
I let here both types stay. Since this is a response dict template, it should contain all the supported fetchable parameters IMO :)